articles/integrations/aws-marketplace.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "SparkPost on AWS Marketplace"
+description: "To explain how to set-up the SparkPost service on AWS"
+---
+
+SparkPost is now offering our email cloud service on the Amazon Web Services (AWS) marketplace. More than just an email API, sophisticated features for applications big and small. Powerful templating, multiple webhook addresses, and multi-account features let you build applications that engage, retain, and rapidly grow your user base.
+
+When you are an AWS marketplace user, you can select Sparkpost as a service and setup a new account. Once you select the pricing option, the marketplace will pass you to our web console to complete the sign-up process. You will have 1 hour from the price selection to complete the sign-up process.
+
+# Pricing
+We have 3 plans available. 15k, 50k and 150k. For more information, please find our page at sparkpost on (AWS Marketplace)
+
+| Pricing option | Hourly cost      | Message volume | Cost per 1000 additional messages |
+|------|----------------------------|----------------|-----------------------------------|
+| 15k  | free                       | 15,000         | N/A | 
+| 50k  | $0.012 (about $9 a month)  | 50,000         | $0.75 |
+| 150k | $0.068 (about $49 a month) | 150,000        | $0.70 |
+
+# Using your new SparkPost account
+
+Please follow our standard process for using your sparkpost account [found here](https://www.sparkpost.com/docs/getting-started/getting-started-sparkpost/).
+
+# Managing your account
+
+You can managing your account on the SparkPost UI or using our APIs. However to unsubscribe, please make this changes on the (AWS Marketplace).
```

Comment:
what's with the quotes? is that supposed to be a link?

Reply:
Yes.